app/views/content/steps-to-become-a-teacher/_step_1_decide_who_to_teach.html.erb

```diff
@@ -8,3 +8,5 @@
 </ul>
 
 <p>Follow these steps if you want to teach primary or secondary, including <a href="/teach-disabled-pupils-and-pupils-with-special-educational-needs">teaching disabled pupils and pupils with special educational needs</a> in these settings.<p>
+
+<p>If you’re an undergraduate, interested in teaching chemistry, computing, languages, maths or physics then a <a href="/teaching-internship-providers">teaching internship</a> might be for you. It lasts for 3 weeks, starts in June and you’ll be paid £300 per week while you experience the classroom.</p>
```

Comment:
```suggestion
<p>If you’re an undergraduate interested in teaching chemistry, computing, languages, maths or physics then you could do a <a href="/teaching-internship-providers">paid teaching internship</a> to find out if teaching is right for you.</p>
```
I just wonder if the 'lasts for 3 weeks, starts in June ... etc.' is a bit too much information for this point? Also makes that para a bit shorter/easier to read. Feel free to completely disagree!